lib/signer.rb

```diff
@@ -196,6 +196,7 @@ def binary_security_token_node
       node['EncodingType'] = 'http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-soap-message-security-1.0#Base64Binary'
       node.content = Base64.encode64(cert.to_der).gsub("\n", '')
       signature_node.add_previous_sibling(node)
+      set_namespace_for_node(node, WSSE_NAMESPACE, ds_namespace_prefix)
```

Comment:
@jeffcarbs Thank you for great fix! My only concern is the regression it might introduce for older Nokogiri setups. 🎉 

Nokogiri provides constant returning semantic version in string format. Here is my suggestion, which executes code if version is equal or higher than `"0.12.0"`.

```suggestion
      set_namespace_for_node(node, WSSE_NAMESPACE, ds_namespace_prefix) if ([1, 12, 0] <=> Nokogiri::VERSION.split('.').map{|v| v.to_i}) <= 0
```

But if is @ebeigarts decision how to proceed.

Reply:
I committed that change, thanks for the suggestion 👍